bin/release.sh

```diff
@@ -36,24 +50,17 @@ npx lerna run build \
   --ignore @bugsnag/plugin-electron-app \
   --ignore @bugsnag/plugin-electron-client-state-persistence
 
+# push latest git tag
+TAG=$(git describe --tags --abbrev=0)
+git push origin $TAG
+
+# publish
+npx lerna publish from-package
```

Comment:
I think we want `from-git` instead of `from-package`?

> bump from-git
>
> &hellip; This will identify packages tagged by lerna version and publish them to npm. This is useful in CI scenarios where you wish to manually increment versions, but have the package contents themselves consistently published by an automated process.

`from-package` still sounds best for `RETRY_PUBLISH` though

Reply:
that's what I initially thought - but I saw in a recent nx walkthrough of lerna they used it this way... let's stick with the documentation though